hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/LockManager.java

```diff
@@ -76,6 +76,7 @@ public void lock() {
             break;
           }
           metrics.updateLockNotAcquiredMetric();
+          LOG.warn("Current lock owner information : " + lockProvider.getCurrentOwnerLockInfo());
           LOG.info("Retrying to acquire lock...");
```

Comment:
would be merged in one line log. `LOG.info("Retrying to acquire lock. Current lock owner information : " + lockProvider.getCurrentOwnerLockInfo())`